src/screens/ChatScreen/components/ChatMessageItem.js

```diff
@@ -232,7 +238,7 @@ const ChatMessageItemComponent = ({ conversation, type, message, created_at, sho
   };
 
   const messageActionModal = useRef(null);
-  const messageActionModalSnapPoints = useMemo(() => [deviceHeight - 680, deviceHeight - 680], []);
+  const messageActionModalSnapPoints = useMemo(() => [deviceHeight - 650, deviceHeight - 650], []);
```

Comment:
@muhsin-k Can you increase the height bit more from 650 to 640 or 630, on android there is a scroll.
```suggestion
  const messageActionModalSnapPoints = useMemo(() => [deviceHeight - 640, deviceHeight - 640], []);
```